super_team_picker/index.js

```diff
@@ -0,0 +1,151 @@
+            const originalRemainder = remainder;
+            if(req.query && req.query.method && req.query.quantity) {
+                if (req.query.method == 'teamCount') {
+                    for (let i = 0; i < Number(req.query.quantity); i++) {
+                        let singleTeam = [];
+                        if (remainder) {
+                            singleTeam = shuffledMembers.slice(i + secondNum * i, i + secondNum * i + secondNum + 1);
+                            remainder--;
+                            finalArr.push(singleTeam);
+                        } else {
+                            singleTeam = shuffledMembers.slice(originalRemainder + secondNum * i, originalRemainder + secondNum * i + secondNum);
+                            finalArr.push(singleTeam);
+                        }
+                    }
+                    //console.log(finalArr);
+                } else {
+                    //if method == 'perTeam'
+                    if (remainder > 0) {
+                        let t = shuffledMembers.slice(0, remainder);
+                        finalArr.push(t)
+                    };
+                    for (let i = 0; i < secondNum; i++) {
+                        let singleTeam = [];
+                        singleTeam = shuffledMembers.slice(remainder + Number(req.query.quantity) * i, remainder + Number(req.query.quantity) * i + Number(req.query.quantity));
+                        //console.log(singleTeam);
+                        finalArr.push(singleTeam);
+                    }
+                    //console.log(finalArr)
+                }
+            }
+
+            res.render('cohorts/show', { 
+                cohort: cohort,
+                method: req.query.method,
+                quantity: req.query.quantity,
+                secondNum: secondNum,
+                finalArr: finalArr
+             })
+        }
+    })
+})
+
+app.get('/cohorts', (req, res) => {
+    knex('cohorts')
+    .orderBy('createdAt', 'DESC')
+    .then(cohorts => {
+        //console.log(cohorts);
+        res.render('cohorts/index', { cohorts: cohorts })
+    })
+})
+
+app.get('/cohorts/:id/edit', (req, res) => {
+    knex('cohorts')
+    .where('id', req.params.id)
+    .first()
+    .then(cohort => {
+        res.render('cohorts/edit', {
+            cohort: cohort
+        })
+    })
+})
+
+app.patch('/cohorts/:id', (req, res) => {
+    knex('cohorts')
+    .where('id', req.params.id)
+    .update({
+        name: req.body.name,
+        logo_url: req.body.logo_url,
+        members_arr: req.body.members_str.split(',')
+    })
+    .then(() => {
+        res.redirect('/cohorts')
+    })
+})
+
+
+
+app.delete('/cohorts/:id', (req, res) => {
+    const { id } = req.params;
+    knex('cohorts')
+    .where('id', id)
+    .del()
+    .then(() => {
+        res.redirect('/cohorts')
+    })
+})
+
+
+app.listen(8080, () => {
+    console.log('ON PORT 8080!')
```

Comment:
just a personal preference, and being super picky here lol I suggest `http://localhost:${process.env.PORT}` or just `http://localhost:8080` if there's no env file.